packages/rulesets/src/spectral/functions/utils.ts

```diff
@@ -90,7 +90,7 @@ export function getProperty(schema: any, propName: string): any {
 }
 
 export function findBodyParam(params: any) {
-  const isBody = (elem: any) => elem.name === "body" && elem.in === "body"
+  const isBody = (elem: any) => elem.name === "body"
```

Comment:
Are we sure we don't need `"in": "body"` for body parameters? It looks like in OpenAPI 2.0 `in` is [required for all parameters](https://swagger.io/specification/v2/#parameter-object), but in OpenAPI 3.0 body parameters use a different property name: `requestBody`.

Reply:
My bad,
The intent was to remove the name===body check